src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -101,16 +127,75 @@ export default class VizShonenJump extends Connector {
             });
     }
 
+    async _getMangaVolumes(manga) {
+        let auth = await this._getUserInfo();
+        const request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request);
```

Comment:
Consider using the CSS query directly `this.fetchDOM(request, '#o_products tr td:last-of-type a')`